modules/deploy/pages/deployment-option/cloud/create-topic.adoc

```diff
@@ -20,7 +20,7 @@ After you create a cluster, you can create a topic for that cluster.
 
 | *Replication factor*
 | The number of partition replicas for the topic. Three replicas are recommended. The value for this property must meet or exceed the cluster's xref:reference:cluster-properties.adoc#minimum_topic_replication[`minimum_topic_replication`] setting. 
-NOTE: Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery. 
+**NOTE:** Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery. 
```

Comment:
'''suggestion

NOTE: Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery.

'''